src/sharedServices/s3/checkS3BucketData.ts

```diff
@@ -10,10 +10,7 @@ import { getAuditDataSourceBucketName } from './getAuditDataSourceBucketName'
 export const checkS3BucketData = async (
   dataRequestParams: DataRequestParams
 ): Promise<S3BucketDataLocationResult> => {
-  logger.info(
-    'Looking for S3 data using params',
-    JSON.stringify(dataRequestParams)
-  )
+  logger.info('Looking for S3 data')
```

Comment:
This log probably a bit useless now without the data it's looking for.

Reply:
Removed now